InterGalacticCurrencyConvertor/src/main/java/numerals/IDecimalConvertor.java

```diff
@@ -0,0 +1,13 @@
+package main.java.numerals;
+
+import java.util.List;
+
+import main.java.IgnConversionException;
+
+/**
+ * Contract that defines a decimal converter.
+ */
+public interface IDecimalConvertor {
+  boolean isValidSymbol(String symbol) throws IgnConversionException;
```

Comment:
Add javadoc to each method in the interface.

Reply:
done